pages/report.tsx

```diff
@@ -0,0 +1,300 @@
+import { Button, ConfigProvider, Form, Layout, Skeleton, Table, Tag, theme } from 'antd';
+import styles from '../styles/Report.module.css';
+import Head from 'next/head';
+import { useSession } from 'next-auth/react';
+import { ArrowLeftOutlined, BugOutlined, CheckCircleFilled, CloseCircleFilled, SendOutlined } from '@ant-design/icons';
+import React, { useEffect, useState } from 'react';
+import TextArea from 'antd/lib/input/TextArea';
+import Link from 'next/link';
+import { RequestType, useCustomSWR } from '../utils/request-utils';
+import { GitHubIssueStatus, Report } from '../types/client';
+import { ColumnsType } from 'antd/lib/table';
+import { Octokit } from 'octokit';
+import { useRouter } from 'next/router';
+
+const ReportBug = () => {
+	const { data: session, status } = useSession();
+	const router = useRouter();
+
+	const [success, setSuccess] = useState<boolean | null>(null);
+	const [loading, setLoading] = useState<boolean>(false);
+	const [githubIssues, setGithubIssues] = useState<GitHubIssueStatus[] | null>(null);
+
+	useEffect(() => {
+		const fetchGithubIssue = async () => {
+			const octokit = new Octokit({
+				auth: process.env.GITHUB_TOKEN,
+			});
+
+			await octokit
+				.request('GET /repos/VandyHacks/witness/issues?state=all', {
+					owner: 'VandyHacks',
+					repo: 'witness',
+					headers: {
+						'X-GitHub-Api-Version': '2022-11-28',
+					},
+				})
+				.then(res => {
+					// parse through issues and match the issue number and status
+					console.log(res.data);
+					const issues = res.data.map((issue: any) => {
+						return {
+							issueNumber: issue.number,
+							status: issue.state,
+						};
+					});
+					setGithubIssues(issues);
+				})
+				.catch(err => {
+					console.error(err);
```

Comment:
call `handleSubmitFailure()` inside of `catch()` and `handleSubmitSuccess()` inside of `then()`